pkg/api/validation/validation.go

```diff
@@ -727,7 +736,7 @@ func ValidateMinionUpdate(oldMinion *api.Node, minion *api.Node) errs.Validation
 
 	if !api.Semantic.DeepEqual(oldMinion, minion) {
 		glog.V(4).Infof("Update failed validation %#v vs %#v", oldMinion, minion)
-		allErrs = append(allErrs, fmt.Errorf("update contains more than labels or capacity changes"))
+		allErrs = append(allErrs, fmt.Errorf("update from %#v to %#v contains more than labels or capacity changes", oldMinion, minion))
```

Comment:
We should also define an actual validation error for this instead of Errorf.  TODO